### PR TITLE
Add drag-and-drop tooltip to events list

### PR DESCRIPTION
**Pull request by maxwofford**
Screen Shot 2022-10-25 at 12 56 18

This link currently goes to example.com, pending a changelog link for the drag-able event ship

### CODE REVIEW

File: app/views/static_pages/index.html.erb [hunk 1]
```diff
@@ -21,6 +21,11 @@
 <main class="flex flex-column items-center center pb3">
   <% if signed_in? %>
     <% if @events.any? %>
+      <% if Flipper.enabled?(:event_reordering_2022_10_10, current_user) %>
+        <%= link_to "https://changelog.bank.hackclub.com/re-order-your-homepage-247413", class: "muted flex items-center justify-end h6", style: "align-self: baseline", target: "_blank" do %>
+          <%= inline_icon "expand", size: 24 %> Drag to reorder
```
Comment (cjdenio) [inline app/views/static_pages/index.html.erb]: I think a mouse pointer icon might be better— maybe [this one](https://fontawesome.com/icons/hand-pointer?s=solid&f=classic) from Font Awesome?

File: app/views/static_pages/index.html.erb [hunk 2]
```diff
@@ -21,6 +21,13 @@
 <main class="flex flex-column items-center center pb3">
   <% if signed_in? %>
     <% if @events.any? %>
+      <% if Flipper.enabled?(:event_reordering_2022_10_10, current_user) %>
```
Comment (cjdenio) [inline app/views/static_pages/index.html.erb]: you could add ` && @events.size > 1` here

File: app/assets/images/icons/hand-pointer-solid.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M128 40c0-22.1 17.9-40 40-40s40 17.9 40 40V188.2c8.5-7.6 19.7-12.2 32-12.2c25.3 0 46 19.5 47.9 44.3c8.5-7.7 19.8-12.3 32.1-12.3c25.3 0 46 19.5 47.9 44.3c8.5-7.7 19.8-12.3 32.1-12.3c26.5 0 48 21.5 48 48v32 64c0 70.7-57.3 128-128 128l-16 0H240l-.1 0h-5.2c-5 0-9.9-.3-14.7-1c-55.3-5.6-106.2-34-140-79L8 336c-13.3-17.7-9.7-42.7 8-56s42.7-9.7 56 8l56 74.7V40zM240 304c0-8.8-7.2-16-16-16s-16 7.2-16 16v96c0 8.8 7.2 16 16 16s16-7.2 16-16V304zm48-16c-8.8 0-16 7.2-16 16v96c0 8.8 7.2 16 16 16s16-7.2 16-16V304c0-8.8-7.2-16-16-16zm80 16c0-8.8-7.2-16-16-16s-16 7.2-16 16v96c0 8.8 7.2 16 16 16s16-7.2 16-16V304z"/></svg>
```
Comment (cjdenio) [inline app/assets/images/icons/hand-pointer-solid.svg]: I actually think the non-solid variant fits Bank better, up to you though

File: app/views/static_pages/index.html.erb [hunk 4]
```diff
@@ -21,6 +21,13 @@
...
+        <% if @events.size > 1 %>
+          <%= link_to "https://changelog.bank.hackclub.com/re-order-your-homepage-247413", class: "muted flex items-center justify-end h6 mb1", style: "align-self: baseline", target: "_blank" do %>
```
Comment (cjdenio) [inline app/views/static_pages/index.html.erb]: What do you think of having this at the bottom of the list instead of the top?

**Review by cjdenio — APPROVED** [review]
This is great! I left a few opinionated comments, but I really like this as-is

File: app/views/static_pages/index.html.erb (hunk 4 above)
Comment (maxwofford) [reply app/views/static_pages/index.html.erb]: Intentionally at the top– I want it to be visible for long event lists. We can drop it to the bottom if we change our mind later!

File: app/assets/images/icons/hand-pointer-solid.svg (hunk 3 above)
Comment (maxwofford) [reply app/assets/images/icons/hand-pointer-solid.svg]: Went with this for consistency with the markdown and dispute icon uses (other instances of floating muted text with icons)
Screen Shot 2022-10-26 at 11 30 56
Screen Shot 2022-10-26 at 11 31 01

Feel free to PR another icon though if it looks better!

File: app/views/static_pages/index.html.erb (hunk 2 above)
Comment (maxwofford) [reply app/views/static_pages/index.html.erb]: Left this in a new line b/c I expect we'll remove the Flipper flag soon (it's now generally available, right?)

**Pull request merged**